src/app/shared/component/content-container/content-container.component.ts

```diff
@@ -12,14 +12,15 @@ import { User } from '@app/models/user.model';
 })
 export class ContentContainerComponent implements OnInit {
 	user: User;
+	public navIsOpened: boolean = false;
```

Comment:
- можно без Is

```suggestion
	public navOpened: boolean = false;
```
- присваивание значения делаем в конструкторе